frontend/src/lib/components/SelectGradientOverflow.tsx

```diff
@@ -40,10 +40,20 @@ function CustomTag({ label, onClose, value }: CustomTagProps): JSX.Element {
 /**
  * Ant Design Select extended with a gradient overlay to indicate a scrollable list.
  */
-export function SelectGradientOverflow(props: SelectProps<any>): JSX.Element {
+export function SelectGradientOverflow({
+    autoFocus = false,
+    defaultOpen = false,
+    delayBeforeAutoOpen = false,
+    ...props
+}: SelectProps<any> & {
+    delayBeforeAutoOpen?: number | false
```

Comment:
I don't think `| false` is needed or meaningful here since `?:` already makes it an optional parameter.

Suggestion:

```
export function SelectGradientOverflow(props: YourType): JSX.Element
```

You can pass `delayBeforeAutoOpen` forward as undefined here.